src/router.ts

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+import * as glob from 'glob'
+import logger from './util/logger';
+import * as path from 'path';
+//key index issue with types:so any
+const router:any = express.Router(); 
```

Comment:
Generally, explicitly typing something from an external library as `any` is indicative of a design pattern problem; was there a reason for this?

Reply:
oh, those can be cleaned up. 
I was having trouble with express, and reading the recommended way I found of importing it was * as express, which I thought was odd.